docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+- [GitHub Actions Reusable Workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+The following tools and concepts are used in `ITKRemoteModuleBuildTestPackageAction`
+and may be helpful or required for contributing:
+
+- [Bash](https://www.gnu.org/software/bash/) and
+  [PowerShell](https://learn.microsoft.com/en-us/powershell/) scripting;
+- [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) build tools;
+- [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) and 
+  [CDash](https://cmake.org/cmake/help/book/mastering-cmake/chapter/CDash.html)
+  testing and reporting tools;
+- [Python](https://www.python.org/) scripting;
+- [Python wheel](https://wheel.readthedocs.io/en/stable/) packaging;
+
+## Creating or Modifying a Workflow
+
+GitHub Actions requires that workflows are placed in the [`.github/workflows`](../.github/workflows) directory.
+This is where you will find existing workflows and possibly add new workflows.
+
+To contribute to `ITKRemoteModuleBuildTestPackageAction` you will need to
+first [fork](https://docs.github.com/en/github-ae@latest/get-started/quickstart/fork-a-repo)
+the repository to your user account on GitHub and then
+[clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+the fork to your local machine.
+
+### Creating a Workflow
+
+`ITKRemoteModuleBuildTestPackageAction` is a central repository for Github Actions
+reusable workflows for ITK external modules. Before creating a workflow, ask yourself
+the following questions:
+1. Does my workflow address a problem for ITK external modules that is not
+   already solved?
+2. Does my workflow provide functionality that will generalize to various ITK external modules?
+3. Does my workflow provide functionality related to continuous integration such as
+   building, packaging, and/or testing steps?
+
+If the answer to all of the above is "yes", then continue on with your new workflow!
+
+Reference
+[Github Actions Reusable Workflow documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow)
+to get started. You will need to:
+1. Create a workflow in the `.github/workflows` folder;
+2. Define workflow inputs;
+3. Define how jobs will be launched;
+4. Define steps that a given job will take.
+
+### Modifying a Workflow
+
+Workflows will need to be maintained and updated over time as tools
+and technology progress. The best approach to modifying a workflow
+is to try to recreate steps on your local system (if possible),
+identify failures and fixes, then translate fixes to the resuable
+workflow in question. If you encounter an issue that requires
+community input you can ask for help on the
+[ITK community Discourse forum](https://discourse.itk.org/).
+
+## Testing Your Changes
+
+Reusable workflows are intended to _manage continuous integration_ for
+ITK external modules in a reusable manner. At the time of writing
```

Comment:
May the "At the time of writing (...)" be split somehow? Looks long/difficult to follow.

Also, is "At the time of writing" complete? i.e. Did you mean "At the time of writing the workflow"? "At the time of writing this (guide)"/"At this time"?

Reply:
Revised to "As of January 2023" for clarity